tools/Vrops.py

```diff
@@ -242,30 +244,32 @@ def get_vms(self, target, token, parent_uuids, vcenter_uuid, query_specs):
             return [], 400
         resourcekind = 'VirtualMachine'
         q_specs = self._set_query_specs(query_specs, resourcekind)
-        amount_vms, api_responding, _ = self.get_latest_stats_multiple(target, token, [vcenter_uuid],
+        amount_vms, http_code, _ = self.get_latest_stats_multiple(target, token, [vcenter_uuid],
                                                                        ['summary|total_number_vms'],
                                                                        'Inventory')
 
         number_of_vms = amount_vms[0].get('stat-list', {}).get('stat', [])[0].get('data', [0])[0] if \
-            api_responding == 200 and amount_vms else 0
+            http_code == 200 and amount_vms else 0
 
         # vrops cannot handle more than 10000 uuids in a single request
         split_factor = int(number_of_vms / 10000)
         if split_factor >= 1:
             uuids_chunked = list(chunk_list(parent_uuids, int(len(parent_uuids) / (split_factor + 1))))
             logger.debug(f'Chunking VM requests into {len(uuids_chunked)} chunks')
             vms = list()
-            api_responding = list()
+            http_codes = list()
+            response_times = list()
 
             for uuid_list in uuids_chunked:
 
-                vm_chunks, api_chunk_responding = self.get_resources(target, token, adapterkind="VMWARE",
+                vm_chunks, http_code, response_time = self.get_resources(target, token, adapterkind="VMWARE",
                                                                      resourcekinds=[resourcekind],
                                                                      uuids=uuid_list, query_specs=q_specs)
                 vms.extend(vm_chunks)
-                api_responding.append(api_chunk_responding)
+                http_codes.append(http_code)
+                response_times.append(response_time)
             logger.debug(f'Number of VMs collected: {len(vms)}')
-            return vms, max(api_responding)
+            return vms, max(http_codes), max(response_times)
```

Comment:
Is `max(response_times)` really a meaningful value here? Technically it will show the longest needed time to get a chunk, most likely the biggest. In the end it would be interesting to see, how long getting all VMs took. I'd rather sum it up here than taking the maximum.

Reply:
That sounds like a good point. I will change that.